diff --git a/doomgeneric/g_game.c b/doomgeneric/g_game.c
index 9954d78..fdc917b 100644
--- a/doomgeneric/g_game.c
+++ b/doomgeneric/g_game.c
@@ -155,7 +155,7 @@ byte		consistancy[MAXPLAYERS][BACKUPTICS];
 
 fixed_t         forwardmove[2] = {0x19, 0x32}; 
 fixed_t         sidemove[2] = {0x18, 0x28}; 
-fixed_t         angleturn[3] = {640, 1280, 320};    // + slow turn 
+fixed_t         angleturn[3] = {1024, 1280, 320};    /* sx-doom-overlay: was 640, faster R-stick slow turn */
 
 static int *weapon_keys[] = {
     &key_weapon1,
